From ec5b7705fe0deb18dd38b3ed4df0ec73dc572147 Mon Sep 17 00:00:00 2001
From: Mikko Levonmaa <mikko.levonmaa@palm.com>
Date: Tue, 19 Feb 2013 11:16:05 -0800
Subject: [PATCH 1/4] xcbcommon should not be a mandatory dependency

Not all platforms that use qtwayland depend on xcbcommon to be present
for them to be functional

This change won't be needed when this commit from dev branch is merged
to stable:
commit 894adf8a447865119be7c6c4d01577d8032f4177
Author: Oswald Buddenhagen <oswald.buddenhagen@digia.com>
Date:   Mon Apr 8 12:54:58 2013 +0200

make failure to configure wayland non-fatal for a top-level build of qt

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 qtwayland.pro | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/qtwayland.pro b/qtwayland.pro
index 3d5a6b0..66dd38a 100644
--- a/qtwayland.pro
+++ b/qtwayland.pro
@@ -11,15 +11,15 @@ qtCompileTest(xcomposite)
 load(qt_parts)
 
 !config_wayland {
-    error(QtWayland requires Wayland 1.0.3 or higher)
+    error("QtWayland requires Wayland 1.0.3 or higher")
 }
 
 !config_xkbcommon {
-    error(QtWayland requires xkbcommon 0.2.0 or higher)
+    warning("No xkbcommon 0.2.0 or higher found, disabling support for it")
 }
 
 !config_wayland_scanner {
-    error(QtWayland requires wayland-scanner)
+    error("QtWayland requires wayland-scanner")
 }
 
 !config_wayland_egl {
-- 
1.8.2.1

